### PR TITLE
Add alluxio actions and introduce alluxio minicluster for test

### PR DESCRIPTION
**Pull request by taojieterry**
1, add alluxio actions
2, introduce alluxio minicluster for test
3, add one unit test for persist action and tests for other actions to be added.

### CODE REVIEW

**Comment by taojieterry** [comment]
The check failed due to added unittest. However this case works on my local environment.
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.smartdata.actions.alluxio.TestAlluxioActions
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
The build has been terminated

File: smart-alluxio/src/test/java/org/smartdata/actions/alluxio/TestAlluxioActions.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package org.smartdata.actions.alluxio;
```
Comment (manuzhang) [inline smart-alluxio/src/test/java/org/smartdata/actions/alluxio/TestAlluxioActions.java]: Missing license header

**Comment by manuzhang** [comment]
@drankye @littlezhou @huafengw

File: smart-alluxio/pom.xml [hunk 2]
```diff
@@ -86,7 +93,16 @@
...
         </dependency>
-
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
```
Comment (huafengw) [inline smart-alluxio/pom.xml]: We already introduced guava, no need to specify version here

Comment (taojieterry) [inline smart-alluxio/pom.xml]: In the main project guava is introduced but the version is 11.0.2 which is not compatible with required version 14.0.1. I am not sure that we can just modify {guava.version} in the main pom.xml.
Also I saw that we specify the guava version in other module such as smart-zeppelin.

**Comment by taojieterry** [comment]
Exception in thread "Thread-1" java.lang.UnsupportedClassVersionError: javax/ws/rs/core/Configurable : Unsupported major.minor version 52.0
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:800)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:449)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:71)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)

It seems that the travis check failed due to jdk version, in travis the run-time jdk version is 1.7, but it supposed to be 1.8 while running alluxio mini-cluster.

File: smart-alluxio/pom.xml (hunk 2 above)
Comment (huafengw) [inline smart-alluxio/pom.xml]: Got it, so let's keep it for now

**Comment by taojieterry** [comment]
I updated JDK from 1.7 to 1.8 in travis to avoid unittest failure. However both jdk1.7 and jdk1.8 works while building this project. Is it OK? @huafengw  @manuzhang

**Comment by littlezhou** [comment]
@taojieterry   It's OK to update JDK version to 1.8, and we also have other modules require 1.8 beside this. @manuzhang @huafengw We have to keep agents compatible with JDK 1.7 as agents most likely will be working with datanodes.

**Pull request merged**